<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Waveform</class>
 <widget class="QWidget" name="Waveform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>波形界面</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QToolBar" name="toolBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color: rgb(228, 228, 228);  /* 设置背景色为灰色 */
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonIconOnly</enum>
     </property>
     <addaction name="action_Open"/>
     <addaction name="action_Add_Event"/>
     <addaction name="action_Start_Acquisition"/>
     <addaction name="action_Stop_Acquisition"/>
     <addaction name="action_Start_Record"/>
     <addaction name="action_bluetooth"/>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>440</y>
        <width>771</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="connectstate">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollBar" name="horizontalScrollBar_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>761</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QCustomPlot" name="customPlot" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>761</width>
        <height>391</height>
       </rect>
      </property>
      <widget class="QScrollBar" name="horizontalScrollBar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>420</y>
         <width>791</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/icons8-opened-folder-64.png</normaloff>:/img/icons8-opened-folder-64.png</iconset>
   </property>
   <property name="text">
    <string>open</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_Add_Event">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/icons8-finish-flag-64.png</normaloff>:/img/icons8-finish-flag-64.png</iconset>
   </property>
   <property name="text">
    <string>add_event</string>
   </property>
   <property name="toolTip">
    <string>点此测试打标</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_Start_Acquisition">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/icons8-play-64.png</normaloff>:/img/icons8-play-64.png</iconset>
   </property>
   <property name="text">
    <string>start_Acquisition</string>
   </property>
   <property name="toolTip">
    <string>开始</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_Stop_Acquisition">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/icons8-stop-64.png</normaloff>:/img/icons8-stop-64.png</iconset>
   </property>
   <property name="text">
    <string>stop_Acquisition</string>
   </property>
   <property name="toolTip">
    <string>停止</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_Start_Record">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/icons8-record-96.png</normaloff>:/img/icons8-record-96.png</iconset>
   </property>
   <property name="text">
    <string>start_Record</string>
   </property>
   <property name="toolTip">
    <string>点此记录波形</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_bluetooth">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/ic_usb_bluetooth.png</normaloff>:/img/ic_usb_bluetooth.png</iconset>
   </property>
   <property name="text">
    <string>_bluetooth</string>
   </property>
   <property name="toolTip">
    <string>刷新设备</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
